packages/luna/src/components/Calendar/v2/Calendar.tsx

```diff
@@ -20,8 +23,18 @@ export const Calendar: React.FC<Props> = ({ date, HourView }) => {
         )}
       >
         <Hours day={date} />
-        <div className="tw-grid tw-grid-cols-7 tw-w-full tw-rounded-tl-3xl">
-          <WeekHours day={date} HourView={HourView} />
+        <div
+          className="tw-grid tw-grid-cols-7 tw-w-full tw-rounded-tl-3xl"
+          style={{
+            height: CALENDER_HEIGHT + "px",
+          }}
+        >
+          {HourView && !DayView ? (
+            <WeekHours day={date} HourView={HourView} />
+          ) : null}
+          {!HourView && DayView ? (
+            <WeekDays day={date} DayView={DayView} />
+          ) : null}
```

Comment:
This logic is a little bit verbose. Let's delegate this conditional logic to the the `WeekDays`. 


```suggestion
        <WeekHours day={date} HourView={HourView} DayView={DayView} />
```

It is more simple.